src/styles/mailchimp.styl

```diff
@@ -5,7 +5,8 @@
   margin: 0 auto
   font-size: 1.5rem
   max-width: 18.75em
-  margin: 0
+  @media $mqMiddle
```

Comment:
more newlines please. It's hard to see where something start or ends when there is no newline. And you get them really cheap too.